### PR TITLE
Change contact email address

### PR DESCRIPTION
**Pull request by dubek**
Modify website and documentation to the new contact email address.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ This is currently supported on Windows only.
 HElayers is provided under a community edition license for non-commercial use;
 see [license](https://ibm.ent.box.com/s/zfl6rt2p09811nyy8yow8t3mpsmkmsw6). For
 commercial deployments and access to the source code, please contact
-[jbuselli@us.ibm.com](mailto:jbuselli@us.ibm.com) for the Premium Edition
+[chamliam@ie.ibm.com](mailto:chamliam@ie.ibm.com) for the Premium Edition
```
Comment (aehud) [inline docs/index.md:53]: you changed Premium to lower case in some places

File: README.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ This is currently supported on Windows only.
 HElayers is provided under a community edition license for non-commercial use;
 see [license](https://ibm.ent.box.com/s/zfl6rt2p09811nyy8yow8t3mpsmkmsw6). For
 commercial deployments and access to the source code, please contact
-[jbuselli@us.ibm.com](mailto:jbuselli@us.ibm.com) for the Premium Edition
+[chamliam@ie.ibm.com](mailto:chamliam@ie.ibm.com) for the Premium Edition
```
Comment (aehud) [inline README.md:57]: here too

File: LICENSE.md [hunk 3]
```diff
@@ -3,6 +3,5 @@
 HElayers is provided under a community edition license for non-commercial use;
 see [license](https://ibm.ent.box.com/s/zfl6rt2p09811nyy8yow8t3mpsmkmsw6). For
 commercial deployments and access to the source code, please contact
-[jbuselli@us.ibm.com](mailto:jbuselli@us.ibm.com) and
-[rohit.panjala@ibm.com](mailto:rohit.panjala@ibm.com) for the Premium Edition
+[chamliam@ie.ibm.com](mailto:chamliam@ie.ibm.com) for the Premium Edition
```
Comment (aehud) [inline LICENSE.md:6]: also here

**Review by aehud — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (dubek) [reply docs/index.md:53]: fixed to change only email addresses.

**Pull request merged**